client/src/plus/bookmarks/index.tsx

```diff
@@ -251,21 +251,20 @@ function BookmarksLayout() {
       </header>
       <Tabs tabs={localeToTab(locale)} />
       <Container>
-        {SELECTED_TAB === TABS.TAB_BOOKMARKS &&
-          (data ? (
-            <>
-              <SearchFilter filters={filters} sorts={sorts} />
+        {SELECTED_TAB === TABS.TAB_BOOKMARKS && (
+          <>
+            <SearchFilter filters={filters} sorts={sorts} />
+            {(data && (
               <DisplayData
                 data={data}
                 isValidating={isValidating}
                 listMutate={listMutate}
                 deleteBookmarked={deleteBookmarked}
                 showEditButton={true}
               />
-            </>
-          ) : (
-            <Loading message="Waiting for data" />
-          ))}
+            )) || <Loading message="Waiting for data" />}
```

Comment:
Loading message could be less technical:

```suggestion
            )) || <Loading message="Fetching bookmarks" />}
```